ar_exercise/exercise.rb

```diff
@@ -51,21 +69,51 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+# apt1 = Apartment.new(address: "123 Lollipop Lane", monthly_rent: 3000, sqft: 800, num_beds: 2, num_baths: 2)
+# apt1.save
```

Comment:
You can use `create` instead of `new` to avoid having to save on a separate line.